File: app/src/main/java/com/example/dzj/myreader/modle/TxtLine.java
package com.example.dzj.myreader.modle;

public class TxtLine {
    private int position;//文章段数
    private int start;//段字符串中的开始位置
    private int end;//结束位置

    public TxtLine(){}

    public TxtLine(int start, int end){
        this.start = start;
        this.end = end;
    }

    public int getPosition() {
        return position;
    }

    public void setPosition(int position) {
        this.position = position;
    }
    public int getStart() {
        return start;
    }

    public void setStart(int start) {
        this.start = start;
    }

    public int getEnd() {
        return end;
    }

    public void setEnd(int end) {
        this.end = end;
    }
}

File: app/src/main/java/com/example/dzj/myreader/modle/TxtLine.kt
package com.example.dzj.myreader.modle

class TxtLine : Cloneable {
    var position: Int = 0//文章段数
    var start: Int = 0//段字符串中的开始位置
    var end: Int = 0//结束位置

    constructor() {}

    constructor(start: Int, end: Int) {
        this.start = start
        this.end = end
    }

    public override fun clone(): Any {
        var txtLine: TxtLine? = null
        try {
            txtLine = super.clone() as TxtLine
        } catch (e: CloneNotSupportedException) {
            e.printStackTrace()
        }

        return txtLine!!
    }
}
